src/validations/rules/sap.sch

```diff
@@ -171,6 +201,37 @@
 
     </sch:pattern>
 
+    <sch:pattern
+        id="assessment-subject">
+        <sch:rule
+            context="oscal:assessment-subject[@type = 'location']">
+            <sch:assert
+                diagnostics="location-not-include-all-element-diagnostic"
+                doc:guide-reference="Guide to OSCAL-based FedRAMP Security Assessment Plans (SAP) §4.3"
+                fedramp:specific="true"
+                id="location-not-include-all-element"
+                role="error"
+                test="not(exists(oscal:include-all))">The FedRAMP SAP references locations individually.</sch:assert>
+        </sch:rule>
+        <sch:rule
+            context="oscal:include-subject[@type='location']">
+            <sch:let
+                name="ssp-locations"
+                value="$ssp-doc/oscal:system-security-plan/oscal:metadata//oscal:location/@uuid ! xs:string(.)" />
+            <sch:let
+                name="sap-locations"
+                value="/oscal:assessment-plan/oscal:metadata//oscal:location/@uuid ! xs:string(.)" />
```

Comment:
**Not** an error, but: is the string coercion necessary? (i.e., should I be doing the same?)